### DISCUSSION TITLE
似乎又被 ban 掉了一位协作者？

### DISCUSSION THREAD

**Discussion by wuliaodexiaoluo** [Meloong-Git/PCL#5144] [讨论]
3gf 的 GitHub Profile 已经 404 了
起源: 阅览 PCL2-1930 时找不到 3gf 的 PR
随后看到 #5139 不明不白的被关闭了，不知道是谁操作
调用 [GitHub API](https://api.github.com/repos/hex-dragon/pcl2/issues/5139) 查询操作日志，确认是 3gf 操作
访问 Profile，404
完结（（（

**Comment by wuliaodexiaoluo** [comment]
~要不然换个平台得了~

**Reply by Deep-Dark-Forest** [reply]
主要得看龙猫愿不愿意

**Comment by WorldHim** [comment]
转自 dc

Thanks for your concern. Last night (CST), my account was suspended immediately after I replied to a comment at https://github.com/Hex-Dragon/PCL2/discussions/4926.

I took a number of steps as soon as I realized this, and have sent a reinstatement request to GitHub Support. Now I'm waiting for GitHub’s response. I also hope that GitHub will lift the restriction on my account.

I am not sure how much time this will take, but I hope all goes well. I will be the first to reply if there are any progress.

EDIT: CST = China Standard Time

**Reply by wuliaodexiaoluo** [reply]
那只能祝他好运了啊（（（
目前为止社区申诉成功的只有我一个人......
